js/hoot/control/conflicts/actions/IdGraphSynch.js

```diff
@@ -47,8 +47,14 @@ Hoot.control.conflicts.actions.idgraphsynch = function (context)
                     callback(f);
                 }
 
-            }
+            } 
         } else {
+            // Check to see if the relation has been deleted.  If so, do not continue...
+            if((_.find(context.history().changes().deleted,{id:fid}))){
+                _validateMemberCnt(fid,null);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["_validateMemberCnt" was used before it was defined](https://www.codacy.com/app/DGIS/hootenanny-ui/file/2565108061/issues/source?bid=3294725&fileBranchId=3294729#l54)